library/core/src/macros/mod.rs

```diff
@@ -313,6 +313,17 @@ pub macro cfg_match {
 ///     }
 /// }
 /// ```
+///
+/// If desired, it is possible to return expressions through the use of surrounding braces.
```

Comment:
```suggestion
/// If desired, it is possible to return expressions through the use of surrounding braces:
```

Nit, colon to introduce an example